ts/features/fci/store/reducers/fciPollFilledDocument.ts

```diff
@@ -0,0 +1,64 @@
+import { getType } from "typesafe-actions";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Millisecond } from "@pagopa/ts-commons/lib/units";
+import { createSelector } from "reselect";
+import { Action } from "../../../../store/actions/types";
+import { fciClearStateRequest, fciPollFilledDocument } from "../actions";
+import { GlobalState } from "../../../../store/reducers/types";
+import { NetworkError } from "../../../../utils/errors";
+
+export const POLLING_FREQ_TIMEOUT = 2000 as Millisecond;
+export const MAX_POLLING_RETRY = 5;
```

Comment:
Wouldn't be better to place this  variables inside the saga file which is using them instead of defining it in the reducer?

Reply:
done [46a98b9](https://github.com/pagopa/io-app/pull/4260/commits/46a98b9785997ea557ef5f83c2c0764f1d627e63)